.luarc.json

```diff
@@ -25,7 +25,8 @@
     },
     "unusedLocalExclude": [ "_*" ],
     "disable": [
-      "luadoc-miss-see-name"
+      "luadoc-miss-see-name",
+      "unused-local"
```

Comment:
This one is actually very helpful when _writing_ code. (luals and luacheck are orthogonal, as they output at different times, so overlap is not harmful and even beneficial.)

Reply:
Ye, true. I guess we'll need to disable each warning separately then.